stable/mongodb-replicaset/values.yaml

```diff
@@ -2,6 +2,10 @@
 nameOverride: ""
 fullnameOverride: ""
 
+# Override the deployment namespace
+# global:
+#   namespaceOverride: ""
```

Comment:
Remove the global and don't comment out `namespaceOverride`, set it to empty string like we do for fullnameOverride and nameOverride.

Reply:
hi @steven-sheehy, I am trying to cover the scenario when this chart is a dependency of another chart and the other chart is the dependency of another one:
```
top-umbrella-chart:
  dependencies:
     - errbit:
       dependencies:
       - mongodb-replicaset
```
with `namespaceOverride` value, to have errbit and its dependencies into a different namespace, we have to override it for all of the dependencies and all of their dependencies:
```
top-umbrella-chart:
  values:
     errbit:
       namespaceOverride: "second-namespace"
     mongodb-replicaset:
         namespaceOverride: "second-namespace"
```
but with `global.namespaceOverride` what is needed is to just override it in parent level and all of the dependencies will follow:
```
top-umbrella-chart:
  values:
     errbit:
       global.namespaceOverride: "second-namespace"

       mongodb-replicaset:
         # global.namespaceOverride: "second-namespace" -- it is not needed anymore, as global values are accessible in child charts, but can be set to the different namespace if needed, 
```
This is a very simple example which can be extended to more complicated one with more layers of dependencies.

From my understanding, The `global` values was introduced to cover this type of scenarios, and namespace is something that I think should be possible to control from top level,

I sent a [message](https://kubernetes.slack.com/archives/C6E3XH1ED/p1583284201036900) in Slack Charts channel as well to see if this is something other people want or not,

Finally, the reason for not setting the namespace is to differentiate between not wanting it override to override it with empty which means using `Release.Namespace` and values from `global` should not be set otherwise it is not possible to override them from top level

Please let me know what you think.

Reply:
Makes sense. Thanks for the explanation.